src/include/iscsi-lio-server/UI_dialogs.rb

```diff
@@ -1,23 +1,21 @@
-require_relative './iscsi-lio-server_helper.rb'
-require_relative './TargetData.rb'
-require 'cwm/widget'
-require 'ui/service_status'
 require 'yast'
+require 'yast2/execute'
+require 'yast2/system_service'
+
 require 'cwm/table'
 require 'cwm/dialog'
-require 'yast2/execute'
+require 'cwm/widget'
+require 'cwm/service_widget'
+
+require_relative './iscsi-lio-server_helper.rb'
+require_relative './TargetData.rb'
```

Comment:
Why `require_relative`? Please, unless it is not possible for some weird reason, use `require`.